modules/installation-operators-config.adoc

```diff
@@ -52,5 +57,43 @@ service-catalog-apiserver            4.4.0     True        False         False
 service-catalog-controller-manager   4.4.0     True        False         False      5m25s
 storage                              4.4.0     True        False         False      5m30s
 ----
+endif::ibm-z[]
+ifdef::ibm-z[]
+----
+$ oc get clusteroperators
```

Comment:
Does this command need watch like the other non-ibm z one above (since the command says "Watch the cluster components come online")?

Reply:
Yep, fixed.